pkg/client/clientset/versioned/scheme/register.go

```diff
@@ -26,13 +26,15 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
 	gatewayv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+	gatewayv1alpha3 "sigs.k8s.io/gateway-api/apis/v1alpha3"
 	gatewayv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
 )
 
 var Scheme = runtime.NewScheme()
 var Codecs = serializer.NewCodecFactory(Scheme)
 var ParameterCodec = runtime.NewParameterCodec(Scheme)
 var localSchemeBuilder = runtime.SchemeBuilder{
+	gatewayv1alpha3.AddToScheme,
```

Comment:
Related to this - we'll need to reference this in several places throughout the codebase to get presubmits to pass. Essentially just need to add v1alpha3 to the lists found here: https://github.com/search?q=repo%3Akubernetes-sigs%2Fgateway-api+a2.AddToScheme&type=code

Reply:
Thank you!